scripts/py_matter_idl/matter_idl/zapxml/handlers/handlers.py

```diff
@@ -347,7 +347,7 @@ class DescriptionHandler(BaseHandler):
     """
 
     def __init__(self, context: Context, target: Any):
-        super().__init__(context)
+        super().__init__(context, handled=HandledDepth.ENTIRE_TREE)
```

Comment:
this should be SINGLE_TAG to detect when we have unexpected nested things inside description. The intent of the parser was to fully notify us if we think we understand something but we actually do not.

Reply:
Looks like https://github.com/project-chip/connectedhomeip/pull/29114 is handling this.

I just copied what events did in terms of HandleDepth...